xmtp_mls/src/storage/encrypted_store/group_message.rs

```diff
@@ -77,6 +86,61 @@ where
     }
 }
 
+#[repr(i32)]
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, FromSqlRow, AsExpression)]
+#[diesel(sql_type = diesel::sql_types::Integer)]
+pub enum ContentType {
+    Unknown = 0,
+    Text = 1,
+    GroupMembershipChange = 2,
+    GroupUpdated = 3,
+}
+
+impl ContentType {
+    pub fn from_string(type_id: &str) -> Self {
```

Comment:
Should always prefer `impl From<String>` rather than hand-rolled, so that generic `Foo: From<String>` types work

Reply:
added https://github.com/xmtp/libxmtp/pull/1435/commits/dc6428cd4633aab9f49ff2408921eed1461b714d